pkg/convert/xliff2.go

```diff
@@ -20,23 +41,34 @@ type xliff2 struct {
 	TrgLang language.Tag `xml:"trgLang,attr"`
 	File    xliff2File   `xml:"file"`
 }
+
 type xliff2File struct {
 	Units []unit `xml:"unit"`
 }
 
 type unit struct {
-	ID     string  `xml:"id,attr"`                  // translation.messages[n].ID
-	Notes  *[]note `xml:"notes>note"`               // Set as pointer to avoid empty <notes></notes> when marshalling.
-	Source string  `xml:"segment>source"`           // translation.messages[n].Message (if no target language is set)
-	Target string  `xml:"segment>target,omitempty"` // translation.messages[n].Message (if target language is set)
-	// No unified standard about storing fuzzy values
+	ID    string  `xml:"id,attr"`    // TODO: translation.messages[n].ID should be based on source text.
```

Comment:
why TODO ?
Message.ID == unit.ID

Message.ID is a identifier it can be source text, like in PO files, but in this case, XLIFF, has their own identifier